`visit:locate --retry` seems broken

Hello everyone, :wave: 

It seems the command to retry to locate old visits doesn't work anymore.
I'm currently using a docker compose setup with image `shlinkio/shlink:3.3.1`.

Here is what I get when issuing `bin/cli visit:locate -r -vvv` after doing `docker exec -it shlink-shlink-1 sh` (`shlink-shlink-1` is the name of my shlink container)

```
/etc/shlink # bin/cli visit:locate -r -vvv

                                                                                                                        
 [INFO] GeoLite2 db file is up to date.                                                                                 
                                                                                                                        

                                                                                                                        
 [ERROR] Shlinkio\Shlink\Core\Visit\Model\VisitType::from(): Argument #1 ($value) must be of type string,               
         Shlinkio\Shlink\Core\Visit\Model\VisitType given                                                               
                                                                                                                        


In ReflectionEnumProperty.php line 93:
                                                                                                                                       
  [TypeError]                                                                                                                          
  Shlinkio\Shlink\Core\Visit\Model\VisitType::from(): Argument #1 ($value) must be of type string, Shlinkio\Shlink\Core\Visit\Model\V  
  isitType given                                                                                                                       
                                                                                                                                       

Exception trace:
  at /etc/shlink/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ReflectionEnumProperty.php:93
 Shlinkio\Shlink\Core\Visit\Model\VisitType::from() at /etc/shlink/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ReflectionEnumProperty.php:93
 Doctrine\ORM\Mapping\ReflectionEnumProperty->initializeEnumValue() at /etc/shlink/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ReflectionEnumProperty.php:77
 Doctrine\ORM\Mapping\ReflectionEnumProperty->setValue() at /etc/shlink/vendor/doctrine/orm/lib/Doctrine/ORM/UnitOfWork.php:2745
 Doctrine\ORM\UnitOfWork->createEntity() at /etc/shlink/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php:266
 Doctrine\ORM\Internal\Hydration\ObjectHydrator->getEntity() at /etc/shlink/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php:492
 Doctrine\ORM\Internal\Hydration\ObjectHydrator->hydrateRowData() at /etc/shlink/vendor/doctrine/orm/lib/Doctrine/ORM/Internal/Hydration/AbstractHydrator.php:196
 Doctrine\ORM\Internal\Hydration\AbstractHydrator->toIterable() at /etc/shlink/module/Core/src/Visit/Repository/VisitRepository.php:73
 Shlinkio\Shlink\Core\Visit\Repository\VisitRepository->visitsIterableForQuery() at /etc/shlink/module/Core/src/Visit/Geolocation/VisitLocator.php:48
 Shlinkio\Shlink\Core\Visit\Geolocation\VisitLocator->locateVisits() at /etc/shlink/module/Core/src/Visit/Geolocation/VisitLocator.php:32
 Shlinkio\Shlink\Core\Visit\Geolocation\VisitLocator->locateVisitsWithEmptyLocation() at /etc/shlink/module/CLI/src/Command/Visit/LocateVisitsCommand.php:114
 Shlinkio\Shlink\CLI\Command\Visit\LocateVisitsCommand->lockedExecute() at /etc/shlink/module/CLI/src/Command/Util/AbstractLockedCommand.php:35
 Shlinkio\Shlink\CLI\Command\Util\AbstractLockedCommand->execute() at /etc/shlink/vendor/symfony/console/Command/Command.php:308
 Symfony\Component\Console\Command\Command->run() at /etc/shlink/vendor/symfony/console/Application.php:1002
 Symfony\Component\Console\Application->doRunCommand() at /etc/shlink/vendor/symfony/console/Application.php:299
 Symfony\Component\Console\Application->doRun() at /etc/shlink/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at /etc/shlink/bin/cli:10

visit:locate [-r|--retry] [-a|--all]
```

I'm not sure how to debug this. I can provide additional data if you'd like.

Can you reproduce this bug ?

Thanks for having a look,
Mickael

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [3.3.2] - 2022-10-18
+### Added
+* *Nothing*
+
+### Changed
+* *Nothing*
+
+### Deprecated
+* *Nothing*
+
+### Removed
+* *Nothing*
+
+### Fixed
+* [#1576](https://github.com/shlinkio/shlink/issues/1576) Fixed error when trying to retry visits location from CLI.
+
+
 ## [3.3.1] - 2022-09-30
 ### Added
 * *Nothing*

--- a/composer.json
+++ b/composer.json
@@ -20,7 +20,7 @@
         "akrabat/ip-address-middleware": "^2.1",
         "cakephp/chronos": "^2.3",
         "doctrine/migrations": "^3.5",
-        "doctrine/orm": "^2.12",
+        "doctrine/orm": "^2.13.3",
         "endroid/qr-code": "^4.4",
         "geoip2/geoip2": "^2.12",
         "guzzlehttp/guzzle": "^7.4",

